tests/integration/defs/accuracy/test_llm_api_pytorch.py

```
@@ -7809,3 +7810,52 @@ def test_nvfp4(self, tp_size, ep_size, mtp_nextn):
+    sampling_params = SamplingParams(max_tokens=MAX_NUM_TOKENS)
+
+    kv_cache_config = KvCacheConfig(free_gpu_memory_fraction=0.6)
+
+    # Inkling emits typed-content channels. Keep the markers through
+    # detokenization and drop the <|content_thinking|> channel, so only the
+    # visible answer is scored.
+    EXTRA_EVALUATOR_KWARGS = dict(
+        post_process_fn=extract_inkling_content,
+        keep_special_tokens=True,
+        preserve_caller_max_tokens=True,
+    )
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)